hubble-be/src/main/java/com/baidu/hugegraph/util/FileUtil.java

```diff
@@ -30,26 +30,27 @@
 
 public final class FileUtil {
 
-    public static long countLines(String path) {
+    public static int countLines(String path) {
         return countLines(new File(path));
     }
 
     /**
      * NOTE: If there is no blank line at the end of the file,
      * one line will be missing
      */
-    public static long countLines(File file) {
+    public static int countLines(File file) {
         if (!file.exists()) {
             throw new IllegalArgumentException(String.format(
                       "The file %s doesn't exist", file));
         }
         long fileLength = file.length();
+        assert fileLength > 0;
         LineNumberReader lineReader = null;
         try {
             FileReader fileReader = new FileReader(file);
             lineReader = new LineNumberReader(fileReader);
-            lineReader.skip(fileLength);
-            return lineReader.getLineNumber();
+            lineReader.skip(fileLength - 1);
+            return lineReader.getLineNumber() + 1;
```

Comment:
add some comments like:
The last character may be a EOL character or a non-EOL character. If it is the EOL, need to add 1 line; if it is the non-EOL, also need to add 1 line, because the next character means the EOF and should also be counted as a line.